flask_resty/api.py

```diff
@@ -16,6 +16,10 @@ def __init__(self, app=None, prefix=''):
             self._app = app
             self.init_app(app)
 
+        # store the view urls for reference. Doesn't support multiple routes
+        # mapped to same view
+        self.views = {}
```

Comment:
I don't think this is safe - plugins should support working w/multiple apps (e.g. see `init_app` below).